components/Post/ArticleCard/index.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from './index.module.css';
+import type { FC } from 'react';
+import type { PostFrontmatter } from '~/types/frontmatter';
+
+type ArticleCardProps = PostFrontmatter & {
+	slug: string;
+};
+
+const ArticleCard: FC<ArticleCardProps> = ({
+	title,
+	description,
+	authors,
+	date,
+	slug,
+}) => (
+	<div className={styles.content}>
+		<h2 className={styles.title}>
+			<a href={`article/post/${slug}`}>{title}</a>
```

Comment:
nit for later: better to extract these into global utilities (you do _not_ want to have to hunt when something changes).